forge-gui/res/cardsfolder/upcoming/river_song.txt

```diff
@@ -0,0 +1,13 @@
+Name:River Song
+ManaCost:1 U R
+Types:Legendary Creature Human Time Lord Rogue
+PT:2/2
+S:Mode$ Continuous | Affected$ You | AddKeyword$ You draw cards from the bottom of your library instead of the top of your library. | Description$ Meet in Reverse — You draw cards from the bottom of your library rather than the top.
+T:Mode$ Scry | ValidPlayer$ Opponent | TriggerZones$ Battlefield | Execute$ TrigPutCounter | TriggerDescription$ Spoilers — Whenever an opponent scries, surveils, or searches their library, put a +1/+1 counter on CARDNAME. Then CARDNAME deals damage to that player equal to its power.
+T:Mode$ Surveil | ValidPlayer$ Opponent | TriggerZones$ Battlefield | Execute$ TrigPutCounter | TriggerDescription$ Spoilers — Whenever an opponent scries, surveils, or searches their library, put a +1/+1 counter on CARDNAME. Then CARDNAME deals damage to that player equal to its power. | Secondary$ True
+T:Mode$ SearchedLibrary | SearchOwnLibrary$ True | ValidPlayer$ Opponent | TriggerZones$ Battlefield | Execute$ TrigPutCounter | TriggerDescription$ Spoilers — Whenever an opponent scries, surveils, or searches their library, put a +1/+1 counter on CARDNAME. Then CARDNAME deals damage to that player equal to its power. | Secondary$ True
```

Comment:
Please put `Secondary$` param before TriggerDescription... description is usually last with shorter information before